### DISCUSSION TITLE
Issues with application testing orders

### DISCUSSION THREAD

**Discussion by danying0123** [etsy/open-api#1214] [Q&A]
According to the Etsy API testing regulations, after publishing the tested listing as active, we created an order for testing the listing with another account (because the store owner cannot create an order for their own products), and the order was automatically cancelled by the system

Subsequently, I set the product to a product name without a test mark and created an order, which was successful. But I am unable to synchronize this order on my application for some reason. Our setting is to synchronize orders that have been paid but not shipped, but they have not been synchronized, and the order status of this order is not displayed in the Etsy backend

Finally, the store we used for testing was detected as abnormal by Etsy and has been suspended. We are unable to create test orders for this store anymore. Our store account is Walruspicnic@gmail.com

I want to know how to test an order? As long as it does not violate regulations and the store will not be closed?

**Comment by danying0123** [comment]
@jcon @capndesign @jonbasebl5 @morria

**Comment by danying0123** [comment]
@Gareth064

**Reply by Gareth064** [reply]
Sorry bud, I can't really relate to your problem. My wife has a very busy Etsy shop so I didn't need to raise test orders whilst building my app as I could just read from her life orders etc.

**Reply by danying0123** [reply]
Okay, thank you. If possible, could you help me with this issue? I would be very grateful!

I would like to inquire about the interface for synchronizing orders that you have set up, as unfortunately we are unable to synchronize orders that have been paid for but have not been shipped; It's was Paid, was Shipped, was Delivered, was Canceled; How to set these four

**Comment by danying0123** [comment]
@nateswart

**Comment by danying0123** [comment]
Our store has been shown as paused. I would like to know if it is possible to find a solution here?
